docs/architecture/raft.rst

```diff
@@ -44,7 +44,12 @@ Enabling Raft
 
     See :doc:`the upgrade guide from 5.2 to 5.4 </upgrade/index>` for details.
 
-ScyllaDB Open Source 5.2 and later, and ScyllaDB Enterprise 2023.1 and later come equipped with a procedure that can setup Raft-based consistent cluster management in an existing cluster. We refer to this as the **Raft upgrade procedure** (do not confuse with the :doc:`ScyllaDB version upgrade procedure </upgrade/index/>`).
+.. warning::
+    Raft is mandatory in ScyllaDB Open Source 6.0 and later, and ScyllaDB Enterprise 2024.2 and later. You cannot prevent Raft from getting enabled upon upgrade to 6.0 or 2024.2.
```

Comment:
Hm, that's the question, how we should write this -- if this PR gets into 5.5
cc @annastuchlik -- please review the last commit (` docs: update after making consistent_cluster_management mandatory `)